eulerpi/core/transformations.py

```diff
@@ -158,6 +195,14 @@ def _calc_gram_determinant(jac: jnp.ndarray) -> jnp.double:
     Returns:
         jnp.double: The pseudo-determinant of the jacobian
 
+    .. code-block:: python
+
+        import jax.numpy as jnp
+        from eulerpi.core.transformations import calc_gram_determinant
+
+        jac = jnp.array([[1,2], [3,4], [5,6], [7,8]])
+        pseudo_det = calc_gram_determinant(jac)
+
```

Comment:
Move the code example to `calc_gram_determinant`? this function should not be called by the user.  Just not sure what happens with the warning in the terminal. the user should know whether inf or 0 was calculated.

Reply:
I am afraid I do not understand your comment regarding the warning.